core-web/libs/edit-content/src/lib/models/dot-edit-content-form.interface.ts

```diff
@@ -1,12 +1,14 @@
 import {
     DotCMSContentTypeField,
     DotCMSContentTypeLayoutRow,
-    DotCMSContentlet
+    DotCMSContentlet,
+    DotCMSWorkflowAction
 } from '@dotcms/dotcms-models';
 
-export interface EditContentFormData {
+export interface EditContentPayload {
     layout: DotCMSContentTypeLayoutRow[];
     fields: DotCMSContentTypeField[];
     contentlet?: DotCMSContentlet;
+    actions?: DotCMSWorkflowAction[];
```

Comment:
Why optional? Don't we need this all the time?

Reply:
Yes, I marked it as optional while I moving things around/refactoring